index.html

```diff
@@ -1,18 +1,20 @@
 <!doctype html>
 <html>
-  <head>
-    <title>Friends of Girl Develop It Columbus!</title>
-    <link rel="stylesheet" type="text/css" href="styles.css">
-  </head>
-  <body>
-     <header>
-        <img src="images/gdic.jpeg" width="180" height="180" alt="GDIC" >
 
-   </header>
-    <main>
+<head>
```

Comment:
👍 
The formatting has been reset.

Reply:
I edited the index file, then checked it in, pushed it and created another pull request.
Was I just supposed to hit the edit button on this page and make the changes here, since it was already being reviewed?